syntax = "proto3";

package blob;

import "google/protobuf/empty.proto";

service BlobService {
  rpc TalkBetweenServices(stream TalkBetweenServicesRequest) returns (stream TalkBetweenServicesResponse) {}
}

// Put

message TalkBetweenServicesRequest {
  string msg = 1;
}

message TalkBetweenServicesResponse {
  string msg = 1;
}
